### ISSUE TITLE
Add definition of "Partner Relationship" and how partner status is granted

### ISSUE THREAD

**Issue by brianjrobertson** [holacracyone/Holacracy-Constitution#90]
I've been working on Issue #77 and #78, and also looking for approaches to reinforce Holacracy's partner vs. employee paradigm, and resist (or at least help highlight) victim meaning-making patterns and projections on the company and other partners.

Well, I've now got a very rough draft of a multi-faceted new system.  It needs more vetting, perhaps some more work, and likely trying it as a constitutional amendment within HolacracyOne for awhile.  But first, I'm posting it here for feedback. I'll attach the draft text below, and here's a [video walkthrough explaining the system](https://drive.google.com/file/d/0BymXxFXb0ha5cUZJUlpvSVExUHc/view?usp=sharing).

**Comment by brianjrobertson** [comment]
## DRAFT SYSTEM BELOW

Partner Pool & Partnership Circle:

The Organization must maintain a pool of Partners to take part in its governance and operations (its "Partner Pool"). To govern and execute on the work of building and maintaining a healthy Partner Pool, the Organization will have a special “Partnership Circle”. The Partnership Circle controls the Organization’s relationships with its Partners, as a Domain. It may add, terminate, govern, or redefine these relationships as it sees fit, within the constraints below.

Partnership Circle Membership:

The Partnership Circle has no Lead Link. Instead, the following parties may cross link into the Circle to help enact its Purpose:
· Any Role that assigns Partners from the Partner Pool into Roles.
· A representative elected by the Partners in the Partner Pool (the “Partnership Link”), to represent the Organization’s need for a healthy pool of Partners available to energize its Roles.

The election of the Partnership Link will use the Integrative Election process, unless the Partnership Circle has defined an alternate mechanism.

All the rules defined in the Constitution for an Anchor Circle with no Lead Link apply to the Partnership Circle as well.

Relationship Contracts:

The relationship between a Partner and the Organization will typically include mutual agreements to entice each party to enter and remain in the relationship, such as agreements around compensation, expected travel, or time a Partner will focus within the organization (altogether the “Relationship Contract”).

Either the Partner or the Organization may modify their Relationship Contract, but only by making a proposal to the other party. If the Partner seeks a change, the proposal must be made to all Roles within the Partnership Circle by default. If a Role in the Circle seeks a change, the person filling that Role must first ensure no other Roles see any Objections to the proposed change before making any agreements with the Partner. In either case, the Partnership Circle may delegate control of Relationship Contracts to a subset of its Roles, in which case those Roles will process any proposed changes on behalf of the Organization instead. Once either party asks for a modified Relationship Contract, the recipient may either accept the proposal or decline by sharing an Objection given their own Purpose or Accountabilities.

Terminating Relationships:

The relationship between a Partner and the Organization may be terminated at any time by either party, without any proposal or consent required of the other party. However, any role wishing to terminate a Partner’s relationship on behalf of the Organization must first gain the authority required to modify a Relationship Contract, as defined above.

If the Relationship Contract between the Organization and a Partner includes restrictions on terminating the relationship, or any other alternate arrangements around termination, those agreements trump these default rules.

Defining Behavioral Agreements:

From time to time, the Organization and its Partners may each desire certain agreements in how Partners will interact with each other or energize their functions as Circle Members of a Circle or Partners of the Organization (each a “Behavioral Agreement”). Behavioral Agreements must be limited solely to behaviors that are generally desired across many Roles or regardless of Roles, and to agreements that are behavioral in nature and don’t promise to achieve specific outcomes beyond the direct control of a Partner.

For a Behavioral Agreement to become a legitimate expectation of a Partner, a Role or Partner desiring that agreement must request it, the Partner gaining the expectation must accept it, and the agreement must be published in a referenceable location. Any party to a Behavioral Agreement may terminate it at any time by giving notice to all affected parties, and striking the agreement from the published record (or requesting an appropriate role do so). No consent of the other party is required to terminate a Behavioral Agreement, unless it’s included in the Relationship Contract.

The Partnership Circle may further adopt a Policy that requires certain Behavioral Agreements as conditions of anyone remaining in or joining the Partner Pool. With such a Policy in place, any Role in the Organization may terminate the relationship with any Partner who does not promptly accept that Behavioral Agreement upon request, or who later withdraws their agreement, with no proposals or further due-process required.

Segmenting the Partner Pool:

The Partnership Circle may adopt a Policy to divide the Partner Pool into defined segments, based on any differentiating criteria the Partnership Circle specifies, and may choose which of its current partner relationships fall within each segment. Once created, a segment behaves like its own separate Partner Pool and gains its own Partnership Circle to manage that pool, using all the rules herein, including the ability to create further sub-segments. A Partnership Circle that creates a segment loses the ability to directly impact any partner relationship within that segment, however it may still define Policies that apply to that segment.

When electing a Partnership Link to a Partnership Circle, the only Partners who may participate are those whose relationship with the Organization is managed by that specific Partnership Circle, and those serving as the elected Partnership Link within a directly-included segment. A Policy of a relevant Partnership Circle may change this default treatment.

When a Partner Pool includes segments, only Roles that assign Partners from a particular segment into Roles may cross link into that segment's Partnership Circle.

**Comment by dienkwik** [comment]
Hi, Brian:

I have several questions and I hope this is the place to ask them:
1. I vote for calling it the "People Pool". Partner has a shareholder feel to it, and not all organizations have people working in the organizations that are shareholders.
2. Where do this type of cross link only circles live in relations to the other "normal" circles, or how are they going to be visualized in GlassFrog? Are they going to be parallel circles to the anchor circle, each with its own special holarchy ? How will the sub segment partnership circles be visualized in relation to 
   the broader partnership circle ? How would the relationship between the subcircle partnership link and the broader circle's partnership link and the fact that they are in fact are filled by the same partner be visualized ?
3. This cross link only circle is really interesting, however, at first glance it does make it a bit more complicated than before, because now you have to deal with parallel holarchies, if the answer to the first question is parallel holarchies.
4. The Partnership Link represents the organization's need for a healthy partner pool. It took me several passes to understand what this means. My initial reaction was why is it not written as representing the partner directly. I suppose you don't want the partnership link to represent the partners individually, but as a collective entity. So the partnership link would never represent, say John in engineering, and bringing his tensions into the governance.  Rather, the partnership link acknowledges that partners such as John may feel the same tension as John, and to ensure healthy available partner pool, it would be a good idea to resolve this tension. Did I get it somewhat right ?
5. "If the partner seeks a change, the proposal must be made to all roles within the partnership circle". This sentence makes it sound like the partner can directly propose changes.  From what I understand, the partner must still go through the partnership link to propose changes.  The Partnership Link may invite the partner to join the meeting, but it doesn't have to. Perhaps, saying something like "if the partner seeks a change, the partner can convey the proposed changes to the partnership link to be brought and resolved in the partnership circle meeting" can make it more obvious ?
6. The content of the behavioral agreement does not need to go through IDM, I take it ? it just needs to be written by a role or a partner and then requested of another partners to be accepted? Is this different from accountability in that it applies to more than one role, to make it easier to make general expectations ?
7. If a behavior agreement is to be made for all circle members, than each of the circle member has to accept the agreement.  Is this correct ?  So,  you can end up with a situation where some circle member agrees to the agreement and have to abide by the agreement,and others don't ?
   If you want to force everyone to accept the behavior agreement, then there has to be a policy stating this as a requirement of being a circle member ?
   Are we not allowed to do the same thing today in version 4.1 through policies ?
8. If I understand it correctly, all partnership links from all sub partnership circles are automatically partnership links for their respective partner pools in the broader partnership circle. Is this correct ? 

That's it for now.

Thanks, 

Dien.

**Comment by julianeroell** [comment]
Brian, are you proposing this as part of a new constitution, or as an app, or as something else?

**Comment by brianjrobertson** [comment]
@martinroell - the goal is to add this to the Holacracy Constitution v5.0, although the current text is written up separately as a constitutional amendment (app) that can be adopted with v4.1, so it can be tested and further refined.

**Comment by brianjrobertson** [comment]
@dienkwik - really appreciate all the great feedback and questions!  I'll process further as soon as I have a chance.

**Comment by julianeroell** [comment]
If this became part of the constitution, we would not adopt it, and probably not advocate Holacracy further: this is _much_ too complicated. The simplicity of "there is a domain on partner relationships" and "the organisation works out how to deal with that" serves us and our network much, much, much better: it is more flexible _and_ more accessible.

(There is really good thinking in this. Could there not be a way to implement this as an app? As a specific way of doing partner relationships?)

**Comment by gudjon** [comment]
Great insights on the Partner Pool. (May I suggest Talent Pool?)

However, I second @martinroell here.  
I have looked at Holacracy as an well optimized OS. -- Linux, rather than Windows. In any case, let's not build Vista. 

This could be a great App for many (some?) organizations to adopt!

**Comment by ebabinet** [comment]
I really like the idea of the Partnership Link existing in all organizations by default. I felt this was a gap in the current constitution and couldn't imagine a scenario where you would not want one.

I second the confusion described in Dien's points 5 & 6. It's not clear to me how a Partner makes a change to a Relationship Contract. It doesn't sound like it is through the Relationship Link, but do they send out a proposal or attend a circle meeting? Is it processed through IDM?

I personally like the term "Working Agreement" better than "Behavioral Agreement", particularly for agreements that are entered into voluntarily. 

I have mixed feelings about "The Partnership Circle may further adopt a Policy that requires certain Behavioral Agreements as conditions of anyone remaining in or joining the Partner Pool. "
This seems very controlling, particularly since such a policy could be adopted at any time. As a Partner, I would want some advance notice and opportunity to provide reaction to any such proposed policy.

**Comment by ebabinet** [comment]
Here's the reason why I prefer Working Agreement over Behavioral Agreement: 

The term Behavioral Agreement is often used with children and students to encourage or prevent certain behavior on trips or at camp, or as a kind of "performance improvement plan" if a student has been misbehaving. It is a one sided contract that the student has to sign, or they can't participate. It is also used in medical contexts to make sure that patients adhere to their treatment plan and don't abuse their medications. So for me, the term has a somewhat negative and coercive connotation.

Working Agreements, on the other hand, is a term that has been used on Agile teams for many years, and refers to the types of voluntary agreements that team members create to establish team norms.

In the case of behaviors that are required to be in the organization (by Policy of the Partnership Circle), the term Working Agreement would not fit, but maybe those are not called Working Agreements and are just Policies or are automatically part of the Relationship Contract.

**Comment by brianjrobertson** [comment]
@martinroell and @gudjon:  While I appreciate that this text _in its current form_ is more complex than we want in the core constitution, I also feel strongly that this general approach or something a lot like it is needed in the core constitution.

It's not just an app, because it’s defining how you adopt people-related apps - i.e. which circle can adopt them and who gets to be included in that membership.  There's also a default treatment for those various functions (hiring, firing, comp changes, etc.) until you adopt an app.  Those defaults can be changed of course with an app, but the part that's more core than app is locating where those changes can be made, and what roles & representatives get to be included in the decision-making.

And, I can tell you from lots of first-hand experience now: many Holacracy adoptions are significantly impaired by not having better defaults here, and by not having an encoded pathway for a partnership-related rep to be able to drive those changes to those defaults.  There is a **_lot_** of tension that these changes will resolve.

So, my current focus is on finding ways to simplify and streamline this as much as possible to mitigate the complexity factor.  And some of that will happen naturally as other new features get implemented - e.g. once we clarify how multi-filled roles make decisions, much of the special-case complexity in this text around that will vanish into that generic approach; and, once we clarify how circles without lead links generically work, some of the special-case text in here covering that will vanish.  But still, that's just a start - I agree that further simplification here will be very useful.

**Comment by brianjrobertson** [comment]
@dienkwik: I'm going through your comments now:

1)  Agreed, I think "People Pool" works better.

2)  Good questions, and I don't know; the GlassFrog team will be tackling these, and I hope they'll be able to come up with some cool approaches to make this simpler via good technology and visualization support.

3)  Yes, it's effectively a parallel holarchy, but this is needed anyway (see issue #61) - there are a bunch of use-cases where two or more roles, circles, or entities might want to partner to form a joint-venture of sorts, with no single lead link or complete containment/ownership by any forming party.

4)  You're spot-on; the goal of the company is to have a healthy partner pool, but not necessarily to satisfy every individual partner's desires - just like in a healthy human-to-human relationship, sometimes you just need to accept someone for who they are rather than try to change them, or accept that it's the wrong relationship if you can't do that and end it.

5)  The intent is to allow partners to directly propose changes to their own relationship agreement in this case; they are representing themselves personally, and the partnership circle is representing the organization, and between the two parties they can agree to whatever makes sense to them, within any constraints acting upon the organization via governance of course.

6)  Yes to all.

7)  Yes to all.

8)  Not quite; the elected partnership link from a sub-pool just gets to participate in the election for a link in the broader pool.

Hope that helps; let me know if it triggers any concerns or feedback...

**Comment by brianjrobertson** [comment]
@ebabinet:  Thanks for the feedback; I'll change it to Working Agreement to avoid that connotation.  I do like the focus on "behavioral", as I think that's a key distinction around what's valid in these agreements; but that's already in the definition/rules around these, so probably doesn't have to be in the label for the term.

**Comment by brianjrobertson** [comment]
@ebabinet:  I also share your mixed feelings about "The Partnership Circle may further adopt a Policy that requires certain Behavioral Agreements as conditions of anyone remaining in or joining the Partner Pool.".  I'm now thinking maybe it works best to switch this around, and explicitly disclaim that any Working Agreements required to join the organization do **_not_** apply to current partners unless they agree to them.  That still leaves the organization the option to terminate someone's partnership if the partner chooses not to agree, but it requires facing that decision more directly and having a conversation, vs. just assuming everyone has implicitly agreed by not leaving.

**Comment by brianjrobertson** [comment]
Thanks again for all of the great feedback on this to everyone on this thread, and to those who spoke to me real-time about it.  I've just completed another pass through my draft document and simplified as much as possible, and also (hopefully) fixed some issues that were raised and improved some weak points.  Here's the latest version; any additional feedback?

---

**Partnership Circle:**

The Organization must build and maintain a pool of Partners to energize its Roles and lead its governance.  To organize that work, the Organization has a special “Partnership Circle”, which controls (as a Domain) the Organization’s side of the relationship with each of its Partners.  The Organization may add, terminate, govern, or redefine these relationships as it sees fit, within the constraints below.

The Partnership Circle sits within the Organization and must support the Organization’s overall Purpose, as defined by the Anchor Circle.  However, the Partnership Circle sits parallel to the Anchor Circle and not as a sub-circle to it, and has no Super-Circle and no Lead Link.  Instead, the Partnership Circle is initially formed of Cross Links.  All of the Constitution’s rules for an Anchor Circle with no Lead Link apply to the Partnership Circle as well, including those allowing the Circle to define its own Purpose.  The Partnership Circle’s Cross Links are:
- A Cross Link for those Roles within the organization that hold the authority to assign Partners to Roles, to represent the need for the Organization to have a requisite pool of Partners to energize its Roles.  The representative filling this Cross Link role will be the Lead Link of the Anchor Circle or, if the Anchor Circle has no Lead Link, then each Lead Link of any immediate Sub-Circles to the Anchor Circle.
- A Cross Link for the Partners of the Organization, to represent the need for the Organization to offer a healthy environment for Partners to contribute to its Purpose.  The representative filling this Cross Link role will be elected by the Partners.
- Any Cross Links to the Anchor Circle who opt-in to representing their Cross Link Role in the Partnership Circle as well.

Each of these representatives has the duties and powers of a Rep Link with regard to the stakeholders they represent.  The Partnership Circle may adopt a Policy to change how the representatives of these Cross Link roles are selected or the number of representatives.  If an election is required to select a representative and no further process is defined, the Facilitator and Secretary may preside over and enact the election using the Integrative Election Process.  However, if the Facilitator deems this process has become stuck, or that the number of participants would make the process impractical, then the Facilitator may instead use any alternate election process the Facilitator believes fair and appropriate.

**Relationship Contracts:**

The relationship between a Partner and the Organization will typically include mutual agreements to entice each party to enter and remain in the relationship, such as agreements around compensation, expected travel, or time a Partner will focus within the organization (altogether the “Relationship Contract”).

Either the Partner or the Organization may modify their Relationship Contract, but only by making a proposal to the other party.  The recipient may then either accept the proposal, or decline by sharing an Objection given their own Purpose or Accountabilities.  The Partnership Circle may define Roles or processes for initiating or processing these proposals on behalf of the Organization.  In the absence of that definition, the rules for impacting a Domain of an Anchor Circle with no Lead Link shall apply, thus giving all Roles in the Circle the opportunity to raise Objections to proposed Relationship Contract changes.

**Terminating Relationships:**

The relationship between a Partner and the Organization may be terminated at any time by either party, without any proposal or consent required of the other party.  If the Relationship Contract between the Organization and a Partner includes restrictions on terminating the relationship, or any other alternate arrangements around termination, those agreements trump these default rules.

**Working Agreements:**

From time to time, the Organization’s Partners may desire certain agreements about how Partners will interact with each other or fulfill their general functions as Partners or Circle Members.  These “Working Agreements” are for shaping the cultural environment and Partner behaviors that underpin Role-based work.  They may only define expectations beyond the scope of what adding an Accountability on a specific Role can achieve, such as defining norms that are relevant across many Roles or regardless of Roles, or shaping when or how a Partner allocates time or energy to a Role.  Working Agreements must focus on concrete acts to do or behavioral constraints to honor; they may not include promises to achieve specific outcomes.

Any Partner may request a Working Agreement of another Partner.  The request becomes a legitimate expectation only if the other Partner accepts the agreement, and only after a relevant party publishes it in a location referenceable to any stakeholder.  A Partner who has made a Working Agreement may terminate it at any time, by giving notice to all affected parties and striking the agreement from the published record (or requesting an appropriate role do so).  This termination does not require the consent of any other party or stakeholder to the agreement, unless the agreement is included in the Relationship Contract.

The Partnership Circle may choose to require acceptance of a Working Agreement as a condition of partnership in the Organization, and thus make it part of the Relationship Contract with all Partners.  However, adopting such a Policy never automatically conveys that agreement upon current Partners, and nor does it automatically terminate a current Partner who declines to accept it.  Even if a Working Agreement is required for new Partners, each existing Partner must still explicitly accept the new agreement before anyone has a legitimate right to expect it of that Partner.

---

**Comment by KoenVeltman** [comment]
great to get the partner context link in the constitution. but is this the simplest form to create this? It reads rather complex. which gives challenges during the initial days of adopting Holacracy in an organization.

For large organizations this is a big hurdle to create a first pilot in part of their organization without needing to change their working relationships. and for small organizations to immediately start with an additional circle where it is already an implementation challenge to start the creates both the tension for me when implementating with both small organizations and part of large organizations that

potential solution: whats the minimum for the constitution? a single partner context link? and add the rest as an App in the Holacracy App store?

**Comment by brianjrobertson** [comment]
@KoenVeltman:  If we just added a single partner context link into the Anchor Circle, that wouldn't solve #77 and #78, unless we also removed the ability to have an Anchor Circle with a Lead Link and mandated that all Anchor Circles use the no-LL rules.  That would reduce some complexity in one part (no extra circle) but add complexity to the Anchor Circle, and I'm not sure that's any better.  Especially when the vast majority of companies practicing Holacracy create a people/partnership circle anyway right now, so it's really not an extra circle, just different rules for running that circle...

**Comment by brianjrobertson** [comment]
One possible way to remove just a little complexity in that last draft implementation is to cut the part about Anchor Circle Cross Links being able to opt-in to also serving in the Partnership Circle.

I actually have no concrete cases where I know that's needed/useful. I just added it because, for constitution updates, I do need to be somewhat concerned about situations I can’t anticipate, so I want evolutions that are “fail safe” - i.e. ones that won’t break effective tension processing, even in scenarios I’m not able to foresee.  Moving a circle out of the main holarchy has the potential to break tension processing if key perspectives have no pathway into that circle to get represented, and I don’t know that the Lead Link route into the Partnership Circle will be sufficient in all cases.  I also don't know that it won’t be, but the more fail-safe option is to allow any Anchor Circle cross link to also show up in the Partnership Circle if desired - that way the Partnership Circle holarchy has all the same potential representation as the main one from the Anchor Circle, and that raises my confidence to near 100% that this change won’t break anything.

That said, I was already at 90%+ confidence before that addition, and perhaps there are other ways to raise the confidence level without adding that extra complexity.  Or perhaps a beta period (planned anyway) for v5.0 will be enough to figure out if there's any missing cases.  Anyone have thoughts on that part and whether it's worth the extra safety vs. complexity to include it vs. cut it?

**Comment by bernardmariechiquet** [comment]
I finally had some time visualizing your walkthrough video and all comments.
First, I must say that I love it - this resolves a lot of tensions we have within iGi and we encounter with our clients regarding all people-related tensions. We've try number of Apps but still weird not using accurately the core elements provided by the Constitution.

When reading the last draft, I was curious to get why you've add the Anchor Circle Cross Links and I sensed a kinda of tension having two parallel circles with no link with the Anchor one. I was envisioning adding a global circle including two kind of holarchies : one for the work and one for the social body/structure - i.e. the whole company including the institution and the community, those two I do think need to be separate as they are different.

And now that I get the why of Anchor Circle Cross Links, it resolve my tension.

I would not take the risk no having these Anchor Circle Cross Links (if they'd like to show up) as the anchor circle (without a Lead Link) would have no way to process tension re the Partnership circle, which would degrade 4.1 capacities. Unless we get a meta structure including the two: institution and social body.

**Comment by oliviercp** [comment]
@brianjrobertson I notice you have also removed the "**Segmenting the Partner Pool**" section - do you still envision that a specific 'partner segment' could adopt collective Working Agreements by defining them at level of the Partnership Circle? It seems that they could, even if it might not be very relevant to the rest of the circle. It certainly cuts on the complexity.

Re: representation from the **Cross Links to the Anchor Circle**, I also don't know if it would be useful, though I also don't think it adds a lot of complexity. It adds some, I suppose, by the simple fact that it makes the Partnership Circle bigger, adding new perspectives to any tension processing, thus adding some complexity. However it doesn't change anything in the process itself, so it doesn't seem like it adds much structural complexity. That said, when thinking about it, did you come up with examples where such a Cross Link would be useful/relevant? I don't see any example.

**Comment by brianjrobertson** [comment]
@ocompagne:

> I notice you have also removed the "Segmenting the Partner Pool" section

Yes, to reduce complexity, and because it's not as necessary given the updated Partnership Circle representation (just one Lead Link representative vs. every Lead Link, so it scales and segmenting isn't necessary).  That said, it's still possible to segment as in the original draft, just via policy now.

> do you still envision that a specific 'partner segment' could adopt collective Working Agreements by defining them at level of the Partnership Circle?

Not sure what you mean by "defining them at the level of the PC"... they could do that simply by agreeing to them with each other.  If they want to require them as a condition of partnership, then they could propose a Partnership Circle policy limiting partnership grants without the agreement, and there's no reason they couldn't have that policy apply to a segment of potential new partners vs. all of them.  So I think I'm saying "yes" to your question, but wanted to verify we're talking about the same things.

> Re: representation from the Cross Links to the Anchor Circle, I also don't know if it would be useful, though I also don't think it adds a lot of complexity. 

Thanks, that's helpful!

>  That said, when thinking about it, did you come up with examples where such a Cross Link would be useful/relevant? I don't see any example.

Not really - just one scenario where it could possibly be needed, but it's highly speculative (and I can't share details here because it involves confidential client stuff).  So it's more a safety precaution to keep it "safe enough to try" vs. anything I know is clearly needed...

**Comment by tylerdanke** [comment]
I was asked what tension I was trying to solve when adopting this as a constitutional amendment. I thought that my notes here might be helpful if contributed here. 
When I show up in the Team Happiness circle I struggled with what role I was representing. I was representing my Board Founder Context role when I was showing up in Team Happiness. I sensed that when we were having Team Happiness circle meetings it was many times uncomfortable and less than ideal to not have at least a Cross Link from Partners representing the voice of the partners. When dealing with employment relationships it is nice to have the voice of the partners, voice of the role assigners, and the voice of the Founder Context all represented. Prior to this amendment We experienced that partners were feeling controlled without representation.

When we are in the Team Happiness circle I had a tension with roles being assigned. Benefits role was assigned to a person who had different ideas on how to energize the role than how the majority of the Team Happiness circle core members sensed would be best. Therefore the Team Happiness circle roles being elected is an improvement and the circle not having a Lead Link is great. The Facilitator is allowed to decide to use a different election process at their discretion. This is appreciated because some roles are a quick everyone agrees that one person should fill and we don't have to take the ten minutes that integrated election takes. 

Relationship Contracts: I didn't exactly have any tension that caused this section. 

Working agreements: We used to have a Team Member role that was being used as the things that we expect every partner to do inside of each of their roles. The Team Member accountabilities became our working agreements. We had a situation this year where we had a partner where they did not want to abide by our Security policies. The working agreement section greatly clarifies how this would go down. 

There is a bit of extra sophistication than the tensions that I sensed but ultimately I think that it was better as a whole than to take pieces away from it at least to try it.

Glassfrog is not up to not removing the rep link assignment. The work around doesn't seem ideal to me at the moment. I appreciate being able to keep the history especially in this circle so I am okay with keeping the expired rep link assignment for the moment.  I actually believe at this moment that saying that the circle does not have a Rep Link is excessively complex, but not enough for me to want to amend the constitution again.

**Comment by gvandegrift** [comment]
I have a clarifying question and a request.

Clarifying question is for this paragraph:

> A Cross Link for those Roles within the organization that hold the authority to assign Partners to Roles, to represent the need for the Organization to have a requisite pool of Partners to energize its Roles. The representative filling this Cross Link role will be the Lead Link of the Anchor Circle or, if the Anchor Circle has no Lead Link, then each Lead Link of any immediate Sub-Circles to the Anchor Circle.

If the anchor circle has a Lead Link, then this means there will only be one Cross Link created?  If the anchor circle has no Lead Link but has three direct Sub-Circles (each with a Lead Link), then there would be three Cross Links?  Those Cross Links are intended to represent all people filling Role-assigning Roles in the organization?

My request:  as part of this, an example of a Relationship Contract and a Working Agreement would be very helpful.  I understand that could vary based on country (or perhaps even state), but without it, I am left to speculate quite a bit.  Companies have things like "policies & procedures", "employee handbooks", and likely many other similar and related documents.  I guess these examples wouldn't live in the constitution, but some sort of supporting documentation would be helpful.

**Comment by denniswittrock** [comment]
I don't have any amendments to this draft to offer. I just wanted to share a different way of dealing with the tension that led to the creation of the Partner Pool amendment: the construct of the [For-Purpose Enterprise](https://medium.com/encode-org/how-to-work-earn-and-live-on-purpose-25c175213030) that encode.org has developed.

It consists of the three containers of **Organization** (ruleset: Holacracy Constitution, focus: the work), **Association** (ruleset: Association Agreement, focus: the people) and **Company** (ruleset: Operating Agreement, focus: legal and financial relationships). The three contexts are represented at the [Anchor Circle level](https://app.glassfrog.com/organizations/1732#circle-6133) with elected crosslinks from our [Association operations circle](https://app.preview.glassfrog.com/organizations/10357#circle-27447).
 I included links to the respective representations on Glassfrog. 

Check out this diagram here for reference:
[https://cl.ly/2W1j1v3g1d3s](https://cl.ly/2W1j1v3g1d3s)

**Comment by brianjrobertson** [comment]
Dennis: I appreciate what you guys are doing, though it's not enough to address the need here... good topic for a dialog real-time sometime.

**Comment by brianjrobertson** [comment]
I'm going with a super scaled down version of this: just enough to give hooks to define more easily via apps in governance.  Basically, I'm just defining what a "Partner Relationship" actually is, and how partnership is granted.
